### DISCUSSION TITLE
Is it possible to move build.sh, build.cmd and build.ps1 to a subfolder?

### DISCUSSION THREAD

**Discussion by codefactors** [nuke-build/nuke#1418] [Q&A]
Adding Nuke to a solution adds 5 entries to the root of the project (the three `build.*` files, the `.nuke` folder and the `build` folder).  On a project that already has a number of necessary files in the root (`.gitignore`, `README.md`, `LICENSE.md`, etc), this means that the content of the `README` file on GitHub is easily pushed off the bottom of the visible page, meaning that people have no idea what the repo is about without scrolling.

Is there any way that the `build.*` files can be moved into a sub-folder, preferably a sub-folder of /build?

**Comment by matkoch** [comment]
Bootstrappers are created in the folder from which you call `nuke :setup`. There's however only a 2-level search. So don't put it into `a/b/c/d/e/`.
